src/utils/authorization.ts

```diff
@@ -0,0 +1,17 @@
+import { AccessLevel, User } from '../user/user.entity';
+import { err, Status } from './communication';
+
+/**
+ * Checks if user access level is administrator.
+ * Returns true if user is admin, and false otherwise.
+ */
+export async function checkUserPrivileges(user: User): Promise<boolean> {
+  if (user.accessLevel === AccessLevel.ADMINISTRATOR) {
```

Comment:
It would be nice to make this function more generic by passing the access level to check in argument